src/proxy/http3/Http3App.cc

```diff
@@ -282,6 +281,17 @@ Http3App::_handle_uni_stream_on_read_ready(int /* event */, VIO *vio)
   }
 }
 
+void
+Http3App::_handle_uni_stream_on_read_complete(int event, VIO *vio)
+{
+  this->_handle_uni_stream_on_read_ready(event, vio);
+
+  QUICStreamVCAdapter *adapter = static_cast<QUICStreamVCAdapter *>(vio->vc_server);
```

Comment:
would it make sense to check for `adapter != nullptr` or maybe an assert?

Reply:
That's already checked in `Http3App::main_event_handler`.